contracts/interchain-token-service/src/flow_limit.rs

```diff
@@ -104,9 +104,15 @@ pub fn set_flow_limit(
         ensure!(limit >= 0, ContractError::InvalidFlowLimit);
     }
 
-    env.storage()
-        .persistent()
-        .set(&DataKey::FlowLimit(token_id.clone()), &flow_limit);
+    if let Some(limit) = flow_limit {
+        env.storage()
+            .persistent()
+            .set(&DataKey::FlowLimit(token_id.clone()), &limit);
```

Comment:
```suggestion
    if let Some(flow_limit) = flow_limit {
        env.storage()
            .persistent()
            .set(&DataKey::FlowLimit(token_id.clone()), &flow_limit);
```
shadowing can be nice to do when the meaning is the same